include/deal.II/lac/trilinos_solver.h

```diff
@@ -114,25 +114,19 @@ namespace TrilinosWrappers
        * base class for all solvers.
        */
       explicit
-      AdditionalData (const bool         output_solver_details   = false,
-                      const unsigned int gmres_restart_parameter = 30);
```

Comment:
Otherwise, every inheritance from SolverBase would need to have this parameter. For SolverCG this doesn't make sense.